sunflower-land/sunflower-land
# [FEAT] Tax free

# Description

This PR supports tax free withdrawals for any SFL that was deposited or earned in marketplace - this is to avoid double taxes.

<img width="484" alt="Screenshot 2024-09-16 at 10 24 11 AM" src="https://github.com/user-attachments/assets/34b0466e-379f-4fd1-863f-d2af82e78a83">

## How to test?

Update your `.env` with the updated Withdrawals contract.

1. Deposit SFL into the game
2. Go to withdraw (you should not be taxxed)
3. Go to withdraw a second time (you should be taxed on any excess)
4. 
## TODO

Deploy + destroy mainnet contraacts
